.travis.yml

```diff
@@ -25,6 +25,7 @@ matrix:
         - COMPOSER='composer-psalm.json'
 install: composer install
 script: make $TEST_SUITE
+dist: trusty
```

Comment:
Fixes build for php 5.4 and 5.5
 
Travis switched some defaults around. See https://travis-ci.community/t/php-5-4-and-5-5-archives-missing/3723/5

Reply:
You can remove this from your branch, I had to make the same change a few days ago in order to merge a different PR